domain/relation/state/relation.go

```diff
@@ -2912,23 +2911,54 @@ WHERE  uuid = $search.uuid
 	return true, nil
 }
 
-// checkExistsByUUID checks if a record with the specified UUID exists in the
-// given table using a transaction and context.
-func (st *State) checkExistsByUUID(
+// checkRelationExistsByUUID checks if a record with the specified relation UUID
+// exists in the given table using a transaction and context.
+func (st *State) checkRelationExistsByUUID(
```

Comment:
question: why replace a generic does it exist method with 2 specific ones doing the same thing?

Reply:
This is a query builder and we've stated multiple times we're not doing that. Also, how ever small, this is a security concern in the long term (the table name wasn't quoted if someone else used it incorrectly, which is a vulnerability waiting to happen). This generic method isn't buying you anything, just duplicate the code.

All query builders now have to be ok'd by Joe.